src/core/payment-status.ts

```diff
@@ -10,10 +10,7 @@ export const MakePaymentStatusChecker = ({
   const getStatus = (): ResultAsync<PaymentStatus, PaymentStatusError> => {
     return decodedInvoice
```

Comment:
shouldn't it be plural `decodeInvoices`?

Reply:
this use case only operates on a _single_ invoice